src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Analytic.hpp

```diff
@@ -0,0 +1,81 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Gauges.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace GeneralizedHarmonic::gauges {
+/*!
+ * \brief Imposes the analytic gauge condition,
+ * \f$H_a=\Gamma_a^{\mathrm{analytic}}\f$.
+ *
+ * \warning Assumes \f$\partial_t \Gamma_a=0\f$ i.e. the solution is static.
+ */
+class Analytic final : public GaugeCondition {
+ public:
+  /// \brief What analytic solution/data to prescribe.
+  struct AnalyticPrescription {
+    static constexpr Options::String help =
+        "What analytic solution/data to prescribe.";
+    using type = std::unique_ptr<evolution::initial_data::InitialData>;
+  };
+
+  using options = tmpl::list<AnalyticPrescription>;
+
+  static constexpr Options::String help{"Apply an analytic gauge condition."};
```

Comment:
Could you change this to `Apply the analytic gauge condition H_a = Gamma_a, where Gamma_a comes from the AnalyticPrescription`.